src/data/resourceToken.js

```diff
@@ -0,0 +1,9 @@
+import redis from 'redis';
+
+const rtokens = redis.createClient();
+
+rtokens.on('error', (err) => {
+    console.log(`Error: ${err}`);
```

Comment:
Could you please add some description about the method that caused the error in the error message for now?